scripts/configure_ncn.sh

```diff
@@ -8,25 +30,36 @@ function enable_and_start() {
     systemctl start $THIS_SERVICE
 }
 
-zypper -n ar --no-gpgcheck https://download.opensuse.org/distribution/leap/15.3/repo/oss opensuse-oss
-zypper addrepo https://download.opensuse.org/repositories/Virtualization:vagrant:Leap:15.3/openSUSE_Leap_15.3/Virtualization:vagrant:Leap:15.3.repo
+zypper -n ar https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/sles-mirror/Backports/SLE-15-SP3_x86_64/standard?auth=basic SUSE-Backports-SLE-15-SP3-x86_64
+zypper -n ar https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/sles-mirror/Products/SLE-Module-Server-Applications/15-SP3/x86_64/product?auth=basic SUSE-SLE-Module-Server-Applications-15-SP3-x86_64-Pool
+zypper -n ar https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/sles-mirror/Updates/SLE-Module-Server-Applications/15-SP3/x86_64/update?auth=basic SUSE-SLE-Module-Server-Applications-15-SP3-x86_64-Updates
+zypper -n ar https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/sles-mirror/Products/SLE-Module-Basesystem/15-SP3/x86_64/product?auth=basic SUSE-SLE-Module-Basesystem-15-SP3-x86_64-Pool
+zypper -n ar https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/sles-mirror/Updates/SLE-Module-Basesystem/15-SP3/x86_64/update?auth=basic SUSE-SLE-Module-Basesystem-15-SP3-x86_64-Updates
+zypper -n --gpg-auto-import-keys refresh
 
-zypper -n refresh
+echo 'Installing KVM packages ... '
 zypper -n install qemu-kvm
 zypper -n install -t pattern \
     kvm_tools \
     kvm_server
+echo 'Installing vagrant-libvirt packages'
 zypper -n install \
     vagrant-libvirt \
-    libguestfs \
```

Comment:
Not sure if we need this anymore given the other changes in this PR, will know after testing.